app/lib/database/utils.ts

```diff
@@ -1,7 +1,15 @@
 import XRegExp from 'xregexp';
+import slug from 'slug';
 
 // Matches letters from any alphabet and numbers
 const likeStringRegex = XRegExp('[^\\p{L}\\p{Nd}]', 'g');
 export const sanitizeLikeString = (str?: string): string | undefined => str?.replace(likeStringRegex, '_');
 
+export const slugifyLikeString = (str?: string) => {
```

Comment:
Can you comment this?
I'm watching this PR from distance, but I still don't understand why you need this.
It's even worse that you're adding a new function here, but you're not updating `utils.test.ts`.
Remember to write code for future us.